cav_msgs/msg/TrailerAngle.msg

```diff
@@ -0,0 +1,13 @@
+#
+# Trailer Angle Message
+# 
+# This message gives the truck trailer angle in radians
+#
+# @author Aswath Suresh
+# @version 0.1
+#
+
+std_msgs/Header header  # timestamp in the header is the time the sensor returned the angle reading
+
+float64 angle #trailer angle in raidans
```

Comment:
Please add a comment describing the + direction. Is it positive to the left or right of centerline facing forward?